src/Propel/Generator/Manager/AbstractManager.php

```diff
@@ -386,30 +386,33 @@ protected function loadDataModels()
      *
      * @throws \Propel\Generator\Exception\BuildException
      *
-     * @return int
+     * @return int number of included external schemas
      */
-    protected function includeExternalSchemas(DOMDocument $dom, $srcDir)
+    protected function includeExternalSchemas(DOMDocument $dom, string $srcDir): int
```

Comment:
Is it a good idea to add the types here? The next release could be another "major" one, otherwise this could be a BC break we might want to avoid.

Reply:
Makes sense. I removed the type hints again